app/forms/mappings/LocalDateFormatter.scala

```diff
@@ -27,6 +27,7 @@ private[mappings] class LocalDateFormatter(invalidKey: String,
                                            dayKey: String,
                                            monthKey: String,
                                            yearKey: String,
+                                           validDateKey: String,
```

Comment:
Better to rename it to invalidDateKey as this key will be used for invalid date.